materials/outline.md

```diff
@@ -0,0 +1,38 @@
+# Entitlement Justice and Measures of Algorithmic Fairness
+
+## Introduction
+ - Summarize importance of algorithmic fairness measures
+ - Introduce the concept of entitlement justice
+ - Introduce argument
+   - Entitlement justice is important in algorithmic fairness, but historically neglected in discussions of fairness measures
+   - (Some/certain) counterfactuals measures emphasize entitlement justice, resolving this neglect
+
+## Entitlement Justice
+ - Present Nozick's theory of entitlement justice
+ - Aknowledge known issues, but extract key points of relevance
+ - Present advantages of entitlement justice (including examples from domains relevant to algorithmic decision makers)
+
+## Algorithmic Fairness Measures
+ - Present formalism of algorithmic fairness measures
+ - Present common measures and hisotry of their devlopment
+   - i.e, why was each measure developed, what does it aim to capture
+
+## Entitlement Justice in Algorithmic Fairness
+ - In what sense are algorithmic decision makers relevant to property decisions
+   - Present college admissions case
+ - How do common fairness measures fail to capture entitlement justice
+    - Present examples from college admissions case
+ - Justification of how entitlement justice would resolve issues with algorithmic decision makers
+   - Include short notes on the psychology of justice associated with algorithmic decisions
+
+## Counterfactual Fairness and Entitlement Justice
+ - To what extent do counterfactual fairness measures capture entitlement justice?
+ - Discuss with issues of counterfactual fairness measures
+   - socially constructed race, gender, etc
```

Comment:
The social constructivist topic is a large one. I need to clearly define how much I plan on addressing it within my argument.